ansible/roles/gpu/tasks/nvidia-gpu-Ubuntu.yaml

```diff
@@ -0,0 +1,29 @@
+---
+# Implements the NVIDIA Driver Installation Quickstart Guide for Ubuntu LTS - see
+# https://docs.nvidia.com/datacenter/tesla/tesla-installation-notes/index.html#ubuntu-lts
+
+- name: install runfile prereqs
+  apt:
+    name:
+      - linux-headers-{{ ansible_kernel }}
+      - build-essential
+      - gcc-12
+      - g++-12
```

Comment:
Do we need new packages in the bundle?

Reply:
no, we don't suppport offline gpu builds of ubuntu